test/integration/data-types.test.js

```diff
@@ -274,3 +274,30 @@ describe(Support.getTestDialectTeaser('DataTypes'), function() {
     testFailure(Type);
   });
 });
+
+describe(Support.getTestDialectTeaser('DataTypes'), function() {
```

Comment:
Isn't this describe the same as the one on the top of the file? Might aswell just remove it.